pom.xml

```diff
@@ -261,6 +317,19 @@
             <groupId>org.sonatype.plugins</groupId>
             <artifactId>nexus-staging-maven-plugin</artifactId>
           </plugin>
+          <plugin>
+            <groupId>com.mycila</groupId>
+            <artifactId>license-maven-plugin</artifactId>
+            <executions>
+              <execution>
```

Comment:
the execution block should be moved to the plugin management section ideally to avoid sub project the duplication.

Reply:
I've tried some combinations and I've found no difference.  As detailed in my other reply, unless I completely redefine executions in each child project, the license plugin either does not execute at all or it executes in each submodule.

Reply:
this is linked to the aggregate / inherited flags.